src/main/java/com/habitmon/domain/habitmon/api/dto/response/HabitMonInfoResponse.java

```diff
@@ -0,0 +1,8 @@
+package com.habitmon.domain.habitmon.api.dto.response;
+
+import com.habitmon.domain.habitmon.domain.HabitMon;
+
+public record HabitMonInfoResponse(
+        HabitMon habitMon
```

Comment:
반환 DTO에 이렇게 객체를 넣기보다는, 필요한 값(필드)만 명시해서 반환해주면 좋을 것 같아요 ㅎ.ㅎ
```suggestion
        String habitMonId,
        String habitMonName..etc
```